### DISCUSSION TITLE
RSS not working for Changelog.md since Repochange

### DISCUSSION THREAD

**Discussion by DSDV** [community-scripts/ProxmoxVE#420] [Q&A] [answered]
Hi all <3 

i love rss to keep track of things. 
So i also had the changelog.md in my feed to see if and which new projects get added. 
since the reposwap i tried to edit the feed url but https://github.com/community-scripts/ProxmoxVE/blob/main/CHANGELOG.md.atom does not work o.o 
it worked with the old repo tho 

is it possible to get this fixed or another way to get a RSS Feed for Changes? 

with kind regards

**Comment by MickLesk** [comment]
Phew, unfortunately I can't tell you anything about that. How did you retrieve it from tteck? This .atom doesn't tell me anything 😅 

I've never worked with RSS feeds before. Maybe, @BramSuurdje ?

**Reply by DSDV** [reply, marked as answer]
_i think it is a github feature https://stackoverflow.com/questions/21986561/rss-to-a-github-file
you "just append" the .atom (its a type of rss feed) and so far worked for every file id like to track (: 

i dont understand why it does not work in this repo :o

for example: https://github.com/tteck/Proxmox/commits/main/README.md.atom_


and while writing i figured out i had a copy paste error o.o 
its a feed for the commits to keep track and i somehow was on the file url. 

sorry for stealing your time 🙈

**Reply by Aloe-recite** [reply]
@DSDV How did you solve it?
https://github.com/community-scripts/ProxmoxVE/blob/main/CHANGELOG.md.atom => 404
The path should be correct. What am I missing?

**Reply by DSDV** [reply]
@Aloe-recite the link was wrong (: 
you have to go to the "changelog" / commit history of a file and append .atom
for example this is the correct link the changelog.mdk: https://github.com/community-scripts/ProxmoxVE/commits/main/CHANGELOG.md
which you can than append .atom for a feed 

its not a nice RSS Feed from the Changelog but at least its a headsup when sth has changed ^^

**Reply by Aloe-recite** [reply]
thanks, still learning my way around GitHub :)

**Reply by DSDV** [reply]
was new for me too when i discovered it on search for another project to get Feeds (: 
if the helperscripts community finds a way to do a "real" update / news feed i would ofc prefer it but have no clue how to do that ^^

**Reply by Aloe-recite** [reply]
For now I'm using this: https://github.com/JcMinarro/ProxmoxVE-Releases
Unofficial but seems to work great!

**Reply by MickLesk** [reply]
We can do the same, every day 23:59 UTC if there are any changes at changelog.md we can create an RELEASE. 

@havardthom is our expert for Github Apps & Workflows 😃

**Reply by Aloe-recite** [reply]
please make in 12 or so...getting notified of a new script release at midnight wouldn't allow me to sleep 😄 
[just joking of course!]

**Reply by MickLesk** [reply]
I specifically said 23.59 UTC :D because most of our contributors are European, and I think that's a good time for the “end of the day”

**Reply by DSDV** [reply]
i think it would be a really nice addition if it does not cost too much maintenance (:
